packages/synapse-react-client/src/components/WizardChoiceButton/WizardChoiceButtonGroup.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Box, BoxProps } from '@mui/material'
+
+export default function WizardChoiceButtonGroup(props: BoxProps) {
```

Comment:
Would love to use `styled('div', ...)` instead but there's no way to set the `role` prop with the `styled` API